extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnWriter.java

```diff
@@ -13,7 +13,7 @@ public interface ColumnWriter extends SafeCloseable {
 
     void addDictionaryPage(Object dictionaryValues, int valuesCount) throws IOException;
 
-    void addPage(Object pageData, int valuesCount) throws IOException;
+    void addPage(Object pageData, int valuesCount, final Class columnType) throws IOException;
```

Comment:
I wonder if we can add the type information to the `ColumnWriter's` constructor, rather than doing this every page?